Fix typos and property name errors in db commands

@claude Please fix 3 simple typos and property name errors in database commands

**Problem:**
Three simple bugs causing compilation errors in database commands.

**Errors to fix:**

**1. db/query.ts line 216 - Typo: `fj` undefined**
```
src/commands/db/query.ts(216,53): error TS2304: Cannot find name 'fj'.
```
Likely should be `flags` based on context.

**2. db/schema.ts line 209 - Missing or incorrect NotionCLIError reference**
```
src/commands/db/schema.ts(209,41): error TS2304: Cannot find name 'NotionCLIError'.
```
Verify import at top of file or fix the reference.

**3. db/update.ts line 116 - Wrong property name**
```
src/commands/db/update.ts(116,31): error TS2339: Property 'data_source_id' does not exist on type '{ database_id: string; }'.
```
The object has `database_id` but code is trying to access `data_source_id`.

**Tasks:**
1. Fix typo `fj` to `flags` in src/commands/db/query.ts line 216
2. Fix NotionCLIError reference in src/commands/db/schema.ts line 209 (check import or fix usage)
3. Change `data_source_id` to `database_id` in src/commands/db/update.ts line 116

**Files to modify:**
- `src/commands/db/query.ts` (fix typo)
- `src/commands/db/schema.ts` (fix NotionCLIError reference)
- `src/commands/db/update.ts` (fix property name)

--- a/src/commands/db/query.ts
+++ b/src/commands/db/query.ts
@@ -203,8 +203,9 @@ export default class DbQuery extends Command {
           // Load from file (new flag or deprecated fileFilter)
           const filterFile = flags['file-filter'] || flags.fileFilter
           const fp = path.join('./', filterFile!)
+          let fj: string
           try {
-            const fj = fs.readFileSync(fp, { encoding: 'utf-8' })
+            fj = fs.readFileSync(fp, { encoding: 'utf-8' })
             filter = JSON.parse(fj)
           } catch (error: any) {
             if (error.code === 'ENOENT') {

--- a/src/commands/db/schema.ts
+++ b/src/commands/db/schema.ts
@@ -13,7 +13,7 @@ import {
   groupExamplesByWritability,
   PropertyExample,
 } from '../../utils/schema-examples'
-import { wrapNotionError } from '../../errors'
+import { NotionCLIError, wrapNotionError } from '../../errors'
 import { resolveNotionId } from '../../utils/notion-resolver'
 
 export default class DbSchema extends Command {

--- a/src/commands/db/update.ts
+++ b/src/commands/db/update.ts
@@ -113,7 +113,7 @@ export default class DbUpdate extends Command {
         ? error
         : wrapNotionError(error, {
             resourceType: 'database',
-            attemptedId: args.data_source_id,
+            attemptedId: args.database_id,
             endpoint: 'dataSources.update'
           })
 

